README.md

```diff
@@ -109,12 +129,11 @@ the server where Jenkins lives will cache these and attempt to reuse them if it
 
 #### CSS/SCSS concepts
```

Comment:
Nick wrote `CSS/SCSS concepts` and I've always been a bit on the fence about it 😏 ... my changes here are just formatting